cfgov/v1/blocks.py

```diff
@@ -163,6 +166,29 @@ class Meta:
         )
 
 
+class AudioPlayer(AbstractMediaChooserBlock):
+    def render_basic(self, value, context=None):
+        if not value:
+            return ''
+
+        if value.type == 'audio':
+            player_code = '''
+            <div>
+                <audio controls preload="media">
+                    {0}
+                    Your browser does not support this audio player.
+                </audio>
+            </div>
+            '''
+        else:
+            return ''
+
+        return format_html(player_code, format_html_join(
+            '\n', "<source{0}>",
+            [[flatatt(s)] for s in value.sources]
+        ))
+
+
```

Comment:
Should we give this an icon? I wish there were an option other than the same icon we use for the video player block, but I couldn't find anything more suitable in the Wagtail UI icon set.

```suggestion
class Meta:
        icon = 'media'
```

Reply:
Whoops, yes! An oversight that I noticed but forgot to correct. Thanks!